x-pack/plugins/features/server/feature_privilege_iterator/sub_feature_privilege_iterator.ts

```diff
@@ -5,10 +5,15 @@
  * 2.0.
  */
 
-import type { KibanaFeature, SubFeaturePrivilegeConfig } from '../../../../../features/common';
-import type { LicenseType } from '../../../../../licensing/server';
+import type { KibanaFeature, SubFeaturePrivilegeConfig } from '../../common';
+import type { LicenseType } from '../../../licensing/server';
 
-export function* subFeaturePrivilegeIterator(
+export type SubFeaturePrivilegeIterator = (
```

Comment:
Nit: you added a TSDoc comment for `FeaturePrivilegeIterator`, but missed it for `SubFeaturePrivilegeIterator` here.